vpr/src/base/vpr_context.h

```diff
@@ -288,6 +288,11 @@ struct RoutingContext : public Context {
 
     std::vector<t_rr_node_route_inf> rr_node_route_inf; /* [0..device_ctx.num_rr_nodes-1] */
 
+    //Information about whether a node is a non-configurable. 1 bit for each node
+    //bit value 0: node is configurable, bit value 1: node is non-configurable
```

Comment:
bit value 0: node is not part of a non-configurable set
bit value 1: node is part of a non-configurable set

(easier to read on two separate lines so I suggest that)